spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -697,6 +697,7 @@ The method `BeanManager.getInterceptionFactory()` returns an `InterceptionFactor
 <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 ----
 
+If type parameter `T` is not a class non-portable behavior results.
```

Comment:
I know what Antoine meant. To me, trying to intercept produced Enum sounds just so bizarre I barely see it worth mentioning. But you might be right.